### PR TITLE
[#30] refactor the whole project

### PR DESCRIPTION
**Pull request by dermatz**
- remove old code
- increase maintainability
- updated readme
- move functions in file `functions`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,53 +31,63 @@ Special thanks to e3n GmbH for your Support!
...
-  build                   Build all themes that are configured in .ddev/config.yaml
-  build -f                Builds all themes without yes/no question (force)
-  build Vendor/theme      Build a specific theme
-  watch Vendor/theme      Watch for CSS and JS changes in a specific theme
+  init, install         Initial Woodoo Setup to read all themes from dataabase and write to .ddev/config-themes.yaml
+  help, -help, -h       Displays this help page
+  version, --version    Show version
+  check, selftest       Frontend Self-Test
+  frontend-update       Update to the latest Version (stable Version)
+  frontend-update -dev  Update to the latest Main-Branch Version (not recommented)
```
Comment (t-muir) [inline README.md:82]:
```suggestion
  frontend-update -dev  Update to the latest Main-Branch Version (not recommended)
```

File: README.md [hunk 2]
```diff
@@ -38,53 +31,63 @@ Special thanks to e3n GmbH for your Support!
...
+  themes                List all available themes that are configured in .ddev/config-themes.yaml
+  build                 Builds all themes that are configured in .ddev/config-themes.yaml
+  build -f              Builds all configured themes without yes/no question
```
Comment (t-muir) [inline README.md:85]:
```suggestion
  build -f              Builds all configured themes without yes/no confirmation
```

File: README.md [hunk 3]
```diff
@@ -38,53 +31,63 @@ Special thanks to e3n GmbH for your Support!
...
+  build theme           Build a specific theme
+  watch theme           Watch for CSS and JS changes in a specific theme
 
 Option:
-  -f                      Builds all themes without yes/no question (force)
-  themecode               Theme-Code from .ddev/config.yaml
+  -f                    Force the build command to run without yes/no question
```
Comment (t-muir) [inline README.md:90]:
```suggestion
  -f                    Force the build command to run without yes/no confirmation
```

File: README.md [hunk 4]
```diff
@@ -4,19 +4,12 @@
 
 ---
 
-https://img.shields.io/github/license/dermatz/ddev-woodoo-buildtools-magento
-
-
 This ddev add-on helps you to manage your theme build processes in Magento total easy. You can run a build command to build all your themes or a specific theme. Don't matter if the theme is a Magento Standard or Custom Theme, or maybe a Hyva theme. It automatically takes care which type of theme you use and will build it for you. If a dependency is missing, Woodoo will install it.
```
Comment (t-muir) [inline README.md:7]:
```suggestion
This ddev add-on helps you to manage your theme build processes in Magento easily. You can run a build command to build all your themes or a specific theme. It doesn't matter if the theme is a Magento Standard Theme, a Custom Theme, or a Hyva theme. It automatically takes care of which type of theme you use and builds it for you. If a dependency is missing, Woodoo will install it.
```

**Review by Morgy93 — CHANGES_REQUESTED** [review]
The recent changes in README.md are not reflected in the help command output and should be updated accordingly.

File: commands/web/woodoo_components/help [hunk 5]
```diff
@@ -26,12 +26,12 @@ if [[ ${found} == true ]]; then
     echo -e "  ${txtcyn}frontend-update -dev${txtrst}  Update to the latest Main-Branch Version (not recommented)"
```
Comment (Morgy93) [inline commands/web/woodoo_components/help:26]:
```suggestion
    echo -e "  ${txtcyn}frontend-update -dev${txtrst}  Update to the latest Main-Branch Version (not recommended)"
```

**Review by Morgy93 — CHANGES_REQUESTED** [review]

**Comment by Morgy93** [comment]
There's a typo in https://github.com/dermatz/ddev-woodoo-buildtools-magento/blob/main/commands/web/woodoo_components/help#L10 you might want to squash as well:

`dataabase` -> `database`

**Review by Morgy93 — APPROVED** [review]

**Pull request merged**